### PR TITLE
Sscsci 1119 iba wording change update

### PR DESCRIPTION
**Pull request by Tim97eng**
### Jira link

See [SSCSCI-1119](https://tools.hmcts.net/jira/browse/SSCSCI-1119)

### Change description

- Updating the wording on evidence provide
- Re-design for iba flow of hearing screens. More sections and wording changes. Reflecting this to CYA page too but no change to what is sent to backend.

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: steps/hearing/the-hearing/TheHearing.js [hunk 1]
```diff
@@ -34,9 +34,9 @@ class TheHearing extends SaveToDraftStore {
 
     return [
       answer(this, {
-        question: this.content.cya.attendHearing.question,
+        question: this.content.cya.attendHearing[`question${this.suffix}`],
         section: sections.theHearing,
-        answer: titleise(content.cya.attendHearing[this.fields.attendHearing.value])
+        answer: titleise(content.cya.attendHearing[this.fields.attendHearing.value + this.suffix])
```
Comment (benouaer) [inline steps/hearing/the-hearing/TheHearing.js:39]: might as well use the backtick for this as well, for consistency

File: steps/reasons-for-appealing/evidence-provide/EvidenceProvide.js [hunk 2]
```diff
@@ -8,12 +8,21 @@ const Joi = require('joi');
...
 const i18next = require('i18next');
+const { isIba } = require('utils/benefitTypeUtils');
 
 class EvidenceProvide extends SaveToDraftStore {
   static get path() {
     return paths.reasonsForAppealing.evidenceProvide;
   }
 
+  /** TODO: Welsh translation for RDN and sentance of welsh MRN */
+  get noticeType() {
```
Comment (benouaer) [inline steps/reasons-for-appealing/evidence-provide/EvidenceProvide.js:19]: should this not be in a content file?

**Review by benouaer — APPROVED** [review]

File: steps/reasons-for-appealing/evidence-provide/EvidenceProvide.js (hunk 2 above)
Comment (Tim97eng) [reply steps/reasons-for-appealing/evidence-provide/EvidenceProvide.js:19]: @benouaer  This is used within the content file to dynamically render the noticeType. Used this method for single text changes

File: steps/hearing/the-hearing/TheHearing.js (hunk 1 above)
Comment (Tim97eng) [reply steps/hearing/the-hearing/TheHearing.js]: @benouaer Tested and pushed. Didn't work when I first tried initially

**Pull request merged**